controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -146,14 +147,25 @@ void client_steering_task::steer_sta()
         TASK_LOG(DEBUG) << "SLAVE " << sta_mac
                         << " has an active socket, sending BACKHAUL_ROAM_REQUEST";
         auto roam_request =
```

Comment:
Since we are in a transition from `vs` to `1905` I think it would be clearer in the code to see the difference in the variables' names. for example -
```suggestion
        auto roam_request_1905 =
```

Reply:
Totaly redundant change suggestion. The only use for it is to check that we did not receive `nullptr`. It could be called as well `cmdu_header`.

Reply:
> Totaly redundant change suggestion

This is a strong statement.
Please read carefully the justification for the change I asked.